tests/publiccloud/migration.pm

```diff
@@ -26,7 +26,9 @@ sub run {
     $self->select_serial_terminal;
     my $provider = $self->provider_factory();
     my $instance = $provider->create_instance();
-    $instance->wait_for_guestregister();
+    $instance->wait_for_guestregister() if is_ondemand();
+    registercloudguest($instance) if (is_byos());
```

Comment:
This line is ok, maybe you can just remove those extra brackets.

```suggestion
    registercloudguest($instance) if is_byos();
```

Reply:
Agree